src/server.ts

```diff
@@ -96,9 +101,11 @@ export class InversifyExpressServer  {
                     router[metadata.method](metadata.path, ...metadata.middleware, handler);
                 });
 
-                this._app.use(controllerMetadata.path, ...controllerMetadata.middleware, router);
+                this._router.use(controllerMetadata.path, ...controllerMetadata.middleware, router);
```

Comment:
Oh, missed this newly created `router`. It should be replaced with custom router too.